packages/usa-banner/src/styles/_usa-banner.scss

```diff
@@ -288,6 +288,12 @@ $banner-icon-close: (
   &[aria-expanded="true"],
   &[aria-expanded="true"]:hover {
     background-image: none;
+
+    @media (forced-colors: active) {
+      &::before {
+        display: none;
+      }
+    }
```

Comment:
### Optional

You could also set `content: none`.
```suggestion
    @media (forced-colors: active) {
      &::before {
        content: none;
      }
    }
```

Reply:
I like this better! Seems less risky

Resolved in [85d12c9](https://github.com/uswds/uswds/pull/5286/commits/85d12c980e65094cb35ee99896b6904305c77c7d)